src/test/scala/is/hail/variant/vsm/GroupBySuite.scala

```diff
@@ -0,0 +1,70 @@
+package is.hail.variant.vsm
+
+import is.hail.SparkSuite
+import is.hail.expr.{TFloat64, TInt64}
+import is.hail.io.annotators.IntervalList
+import org.testng.annotations.Test
+import is.hail.TestUtils._
+import is.hail.utils._
+
+
+class GroupBySuite extends SparkSuite {
```

Comment:
regarding the name, does this mean you are planning to put GroupSamplesBy tests here too? I don't have strong opinion

Reply:
yeah, I think I'm going to do that since splitting them up seems like a lot